tests/dummy/config/optional-features.json

```diff
@@ -0,0 +1,3 @@
+{
+  "jquery-integration": true
```

Comment:
`true` here was necessary because `ember-mocha` depends on the global jQuery object being available (the default is `false` and it should be returned to that at some point in the future)